src/CMakeLists.txt

```diff
@@ -44,3 +70,61 @@ endif()
 if(ENABLE_OV_TF_FRONTEND)
     add_dependencies(inference_engine tensorflow_ov_frontend)
 endif()
+
+# install TBB
+
+# define variables for InferenceEngineConfig.cmake
+# if(THREADING MATCHES "^(TBB|TBB_AUTO)$")
+#     set(IE_TBB_DIR "${TBB_DIR}")
+#     list(APPEND PATH_VARS "IE_TBB_DIR")
+# endif()
+
+# if(install_tbbbind)
+#     set(IE_TBBBIND_DIR "${TBBBIND_2_5}")
+#     list(APPEND PATH_VARS "IE_TBBBIND_DIR")
+# endif()
+
+# install only downloaded TBB, system one is not installed
+if(THREADING MATCHES "^(TBB|TBB_AUTO)$" AND TBBROOT MATCHES ${TEMP})
+    ie_cpack_add_component(tbb REQUIRED)
+    ie_cpack_add_component(tbb_dev REQUIRED)
+    list(APPEND core_components tbb)
+    list(APPEND core_dev_components tbb_dev)
+
+    install(DIRECTORY "${TBB}/lib"
+            DESTINATION runtime/3rdparty/tbb
+            COMPONENT tbb)
+    # Windows only
+    if(EXISTS "${TBB}/bin")
+        install(DIRECTORY "${TBB}/bin"
+                DESTINATION runtime/3rdparty/tbb
+                COMPONENT tbb)
+    endif()
+    install(FILES "${TBB}/LICENSE"
+            DESTINATION runtime/3rdparty/tbb
+            COMPONENT tbb)
+
+    # set(IE_TBB_DIR_INSTALL "3rdparty/tbb/cmake")
+    install(FILES "${TBB}/cmake/TBBConfig.cmake"
+                  "${TBB}/cmake/TBBConfigVersion.cmake"
+            DESTINATION runtime/${IE_TBB_DIR_INSTALL}
+            COMPONENT tbb_dev)
+    install(DIRECTORY "${TBB}/include"
+            DESTINATION runtime/3rdparty/tbb
+            COMPONENT tbb_dev)
+endif()
+
+if(install_tbbbind)
+    install(DIRECTORY "${TBBBIND_2_5}/lib"
+            DESTINATION runtime/3rdparty/tbb_bind_2_5
+            COMPONENT tbb)
+    install(FILES "${TBBBIND_2_5}/LICENSE"
+            DESTINATION runtime/3rdparty/tbb_bind_2_5
+            COMPONENT tbb)
+
+    # set(IE_TBBBIND_DIR_INSTALL "3rdparty/tbb_bind_2_5/cmake")
+    install(FILES "${TBBBIND_2_5}/cmake/TBBBIND_2_5Config.cmake"
+            DESTINATION runtime/${IE_TBBBIND_DIR_INSTALL}
+            COMPONENT tbb_dev)
+endif()
```

Comment:
these rules are for TBB installation. Can we move them under `src/cmake/tbb_install.cmake`

Reply:
Fixed